app/lib/frontend/model_properties.dart

```diff
@@ -9,6 +9,15 @@ import 'dart:convert';
 import 'package:gcloud/db.dart';
 import 'package:yaml/yaml.dart';
 
+Map<String, dynamic> _loadYaml(String yamlString) {
+  final Map map = loadYaml(yamlString);
+  // TODO: remove this part after yaml returns a proper map
+  if (map is YamlMap) {
```

Comment:
Can yaml contain a list that contains a map? If so we also want to do it in that case.

Reply:
I believe it can never be the case with `pubspec.yaml`, and the `pub` client validation and/or the upload will fail if it is not a Map.